traits/observers/_dsl_grammar.lark

```diff
@@ -0,0 +1,54 @@
+// Grammar for Traits Mini Language used in observe
```

Comment:
Just realized nowhere in the file says what this lark file is for (it is kind of implicitly there in the filename/context). So I added this line.

Reply:
Could we also record somewhere the exact command-line command needed to generate the parser from the grammar? (We probably eventually want that wrapped up in an `etstool` task, too, but that's separate.)

Reply:
Sorry, ignore. I should have looked harder at the PR first. (/me crawls back into his hole)

Reply:
Not at all, you helped me realize that I misspelt the filename for `etstool.py` as `etstools.py` on line 3. Now fixed.  Waiting for CI...